pkgs/by-name/sl/slimevr/package.nix

```
@@ -0,0 +1,136 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  replaceVars,
+  makeWrapper,
+  slimevr-server,
+  nodejs,
+  pnpm_9,
+  rustPlatform,
+  cargo-tauri,
+  wrapGAppsHook3,
+  pkg-config,
+  openssl,
+  glib-networking,
+  webkitgtk_4_1,
+  gst_all_1,
+  libayatana-appindicator,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "slimevr";
+  version = "0.13.2";
+
+  src = fetchFromGitHub {
+    owner = "SlimeVR";
+    repo = "SlimeVR-Server";
+    rev = "v${version}";
+    hash = "sha256-XQDbP+LO/brpl7viSxuV3H4ALN0yIkj9lwr5eS1txNs=";
+    # solarxr
+    fetchSubmodules = true;
+  };
+
+  buildAndTestSubdir = "gui/src-tauri";
+
+  # `hyper-tls` and `rustls-pemfile` aren't in the upstream lockfile, but are in
+  # Tauri's. This lock file is the upstream one with both of those added.
+  # `hper-tls` and `rustls-pemfile` are optional dependencies of `reqwest`.
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+
+  pnpmDeps = pnpm_9.fetchDeps {
+    pname = "${pname}-pnpm-deps";
+    inherit version src;
+    hash = "sha256-5IqIUwVvufrws6/xpCAilmgRNG4mUGX8NXajZcVZypM=";
+  };
+
+  nativeBuildInputs = [
+    nodejs
+    pnpm_9.configHook
+    cargo-tauri.hook
+    pkg-config
+    wrapGAppsHook3
+    makeWrapper
+  ];
+
+  buildInputs =
+    [
+      openssl.dev
```

```suggestion
      openssl
```
Unless I am mistaken `.dev` is the default for buildInputs.